onlineapi/src/main/kotlin/com/codingblocks/onlineapi/CustomResponseInterceptor.kt

```diff
@@ -0,0 +1,55 @@
+package com.codingblocks.onlineapi
+
+import okhttp3.Interceptor
+import okhttp3.Request
+import okhttp3.Response
+import org.omg.CORBA.Object
```

Comment:
why is CORBA imported ? it will not work on old versions of Android

Reply:
I accidentally imported it i guess